bedrock/privacy/templates/privacy/faq.html

```diff
@@ -206,8 +206,8 @@ <h2>
             {%- endtrans -%}
           </p>
           <p>
-            {%- trans -%}
-              You don't need an account to use Firefox.  Accounts are required to sync data across devices, but we only ask you for an email address.  We don't want to know things like your name, address, birthday and phone number.
+            {%- trans accounts=url('firefox.accounts') -%}
```

Comment:
@alexgibson @craigcook We need a tag to update this string. This page is activated for quite a few locales: https://l10n.mozilla-community.org/langchecker/?locale=all&website=0&file=privacy/faq.lang

Reply:
Sorry I forgot this. Added the tag `firefox-accounts-112018`